package.nls.json

```diff
@@ -384,7 +385,9 @@
     "AWS.output.sam.no.image.support": "Debugging Image-based Lambdas requires a minimum SAM CLI version of 1.13.0.",
     "AWS.output.sam.local.no.go.support": "Debugging go1.x lambdas requires a minimum SAM CLI version of {0}. Function will run locally without debug.",
     "AWS.output.sam.local.no.net.3.1.debug": "Debugging dotnetcore3.1 lambdas requires a minimum SAM CLI version of 1.4.0. Function will run locally without debug.",
+    "AWS.output.sam.local.no.arm.net.3.1.debug": "The vsdbg debugger does not currently support the arm64 architecture. Function will run locally without debug.",
     "AWS.output.building.sam.application.complete": "Build complete.",
+    "AWS.output.sam.invalidArchitecture": "Invalid architecture specified in {0}. Defaulting to x86_64 architecture for invocation.",
```

Comment:
these messages aren't need in package.nls.json unelss they are used in package.json